consensus/istanbul/backend/announce.go

```diff
@@ -226,12 +355,24 @@ func (sb *Backend) handleIstAnnounce(payload []byte) error {
 		return errUnauthorizedAnnounceMessage
 	}
 
+	// Decrypt the EnodeURL
+	encryptedEnodeData := msg.EncryptedEnodeData
+	var encryptedEnodeUrls map[common.Address][]byte
+	json.Unmarshal(encryptedEnodeData, &encryptedEnodeUrls)
+	encryptedEnodeUrl := encryptedEnodeUrls[sb.Address()]
```

Comment:
What happens if `encryptedEnodeUrl` is nil?

Reply:
the decrypted result should be set to an empty slice